### PR TITLE
<fix>(lombok): add lombok @Data & fix @Setter

### PR DESCRIPTION
**Pull request by xiaoma20082008**
just see the code

### CODE REVIEW

**Review by iocanel — APPROVED** [review]
LGTM

**Comment by xiaoma20082008** [comment]
@iocanel 
Please review again and I had added tests and fixed docs, thank you.

File: examples/builder/hello-lombok/src/main/java/io/sundr/examples/World.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ **/
+
+package io.sundr.examples;
+
+import lombok.Data;
+import io.sundr.builder.annotations.Buildable;
+
+@Buildable
+@Data
+public class World {
```
[the remark this answers is not in the document]
Comment (xiaoma20082008) [reply examples/builder/hello-lombok/src/main/java/io/sundr/examples/World.java]: ok, renamed to `User`

**Review by iocanel — APPROVED** [review]
LGTM

**Pull request merged**